src/components/PanelV2/_mixins.scss

```diff
@@ -140,7 +140,7 @@
     bottom: 0;
     width: $panel__sizing__width;
 
-    .#{$namespace}button {
+    .#{$namespace}button:not(.#{$namespace}combo-button__overflow-menu) {
```

Comment:
@j1mie that would likely only catch situations where someone uses the deprecated `primaryAction` prop or renders a `ComboButton` inside the new `renderFooter` render prop. 🤔 

Since any button could be rendered via `renderFooter`, it seems best to just filter out the one instance where we don't want that flex rule to apply.